src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    public Bulldozer(String machine) {
+        super(machine);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(machine + " started its work.");

```

Comment:
The message should indicate what type of `Machine` is working right now. Consider specifying that it is a `Bulldozer` that started its work, to make the message more informative.